### ISSUE TITLE
Screen reader: The delete screenshot button is not read

### ISSUE THREAD

**Issue by rleitan** [mozilla/webcompat-blipz-experiment#179]
[Affected versions]:
Firefox Beta Version 64.0b6

[Affected Platforms]:
Windows 10

[Prerequisites]:
Enable NVDA screen reader: https://www.nvaccess.org/
Add "extensions.webcompat-blipz-experiment.variation
" pref with "more-sentiment" value in about:config.
The webcompat_blipz_experiment@shield.mozilla.org-2.0.1-signed.xpi is installed.

[Steps to reproduce]:
1.Open the browser with the profile from prerequisites
2.Navigate to a website from the Blipz Sites list and wait for the doorhanger
3.Go to It's slow or I'm having problems
4.Take a screenshot
5.Hover using mouse over the screenshot added to the report
6.Hover over the red trash button

[Expected result]:
The delete screenshot button should be read by the screen reader

[Actual result]:
The delete screenshot button is not read

**Comment by rleitan** [comment]
Also, the back arrow button located near Problem Report is not read

**Comment by miketaylr** [comment]
Thanks, we'll clean up a11y and related concerns in the next version.
